locations/spiders/hobby_lobby.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urlparse
+
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class HobbyLobbySpider(SitemapSpider, StructuredDataSpider):
+    name = "hobby_lobby"
+    allowed_domains = ["hobbylobby.com"]
```

Comment:
Ideally I think this should be:

`allowed_domains = ["www.hobbylobby.com"]`

(e.g. restrict to a specific subdomain `www` that is being used)